packages/client/package.json

```diff
@@ -61,7 +61,7 @@
     "finsemble:build": "cross-env TARGET=finsemble vite build",
     "storybook": "storybook dev -p 6006",
     "storybook:build": "cross-env STORYBOOK=true storybook build -o ./dist/storybook",
-    "_e2e:openfin:run": "cross-env-shell of-automation $npm_config_manifest_url ./e2e/openfin.spec.js --chromeDriverOverride=122 --devToolsPort=9091 --closeRuntime=never",
+    "_e2e:openfin:run": "cross-env-shell of-automation $npm_config_manifest_url ./e2e/openfin.spec.js --chromeDriverOverride=116 --devToolsPort=9091 --closeRuntime=never",
```

Comment:
forgot to adjust the chromedriver version again .. this is a pain, so I might comment the OF version numbers in vite.config.ts unless there is a way of syncing them up (I couldn't find one before)

Reply:
fixed now .. removing the override is now working ok (it was certainly not previously), taking the chromium version from the specified runtime version